src/main/java/de/tum/in/www1/artemis/domain/exam/monitoring/actions/HandedInEarlyAction.java

```diff
@@ -0,0 +1,14 @@
+package de.tum.in.www1.artemis.domain.exam.monitoring.actions;
+
+import javax.persistence.DiscriminatorValue;
+import javax.persistence.Entity;
+
+import de.tum.in.www1.artemis.domain.exam.monitoring.ExamAction;
+
+/**
+ * This action indicates whether a student wants to hand in early or not.
```

Comment:
Considering that there is no additional data in this action, I think that this comment is not really accurate. Something like "indicates that the student handed in early" would make more sense imo.